sepsis-quiz-js/src/sepsisQuiz.js

```diff
@@ -116,11 +119,6 @@ class SepsisQuiz {
       }
     }
 
-    /* true or false */
```

Comment:
did this cause one of the bugs? It would be nice to have this to future proof this for them